### ISSUE TITLE
Fix skip link issue on News page

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#2056]
Check failure on line 7 in frontend/playwright-tests/pageRouteNews.spec.ts

GitHub Actions
/ Runs frontend E2E
[E2E] › pageRouteNews.spec.ts:4:1 › News Tab Loads
  1) [E2E] › pageRouteNews.spec.ts:4:1 › News Tab Loads ────────────────────────────────────────────
    Error: expect(received).toPassAxe(expected)

    Expected: No violations
    Received: 1 violations

    Violations: skip-link(1)

      5 |     await page.goto(NEWS_TAB_LINK, { waitUntil: "networkidle" });
      6 |     // @ts-ignore
    > 7 |     await expect(page).toPassAxe()
        |                        ^
      8 |     await expect(page.getByRole('heading', { name: 'News and Stories', exact: true })).toBeVisible();
      9 | });

        at file:///home/runner/work/health-equity-tracker/health-equity-tracker/frontend/playwright-tests/pageRouteNews.spec.ts:7:24
    await expect(page.getByRole('heading', { name: 'News and Stories', exact: true })).toBeVisible();
